app/jobs/regular/detect_translate_post.rb

```diff
@@ -20,6 +20,8 @@ def execute(args)
 
         begin
           DiscourseTranslator::PostTranslator.translate(post, locale)
+        rescue FinalDestination::SSRFDetector::LookupFailedError
+          # do nothing, there are too many sporadic lookup failures
```

Comment:
Do we want to consider queuing the job again instead with some delay? I'm curious what happens if we do nothing here.

Reply:
Thank you for the clarification!